### PR TITLE
Add an example of HTTPS with a non-default port

### PR DESCRIPTION
**Pull request by bemasc**
Suggested by @sftcd

### CODE REVIEW

**Review by Habbie — APPROVED** [review]

File: draft-ietf-dnsop-svcb-https.md [hunk 1]
```diff
@@ -1212,12 +1212,14 @@ Consider a simple zone of the form:
 
 The domain owner could add this record:
 
-    simple.example. 7200 IN HTTPS 1 . alpn=h3
+    @ 7200 IN HTTPS 1 . alpn=h3
 
-to indicate that simple.example uses HTTPS, and supports QUIC
+to indicate that https://simple.example uses HTTPS, and supports QUIC
```
Comment (enygren) [inline draft-ietf-dnsop-svcb-https.md:1217]:
```suggestion
to indicate that https://simple.example (and http://simple.example)
use HTTPS, and supports QUIC
```

File: draft-ietf-dnsop-svcb-https.md [hunk 2]
```diff
@@ -1212,12 +1212,14 @@ Consider a simple zone of the form:
...
 in addition to HTTPS over TCP (an implicit default).
-The record could also include other information
-(e.g. non-standard port, ECH configuration).
+The record could also include other information (e.g. non-standard port,
+ECH configuration).  For https://simple.example:8443, the record would be:
+
+    _8443._https 7200 IN HTTPS 1 . alpn=h3
```
Comment (enygren) [inline draft-ietf-dnsop-svcb-https.md:1223]: Corner-case that I'm not sure we cover.  What happens to http://simple.example:8443 in this case?  Does it also get forced to HTTPS?

File: draft-ietf-dnsop-svcb-https.md (hunk 1 above)
Comment (bemasc) [reply draft-ietf-dnsop-svcb-https.md]: I added a sentence about this at the end of the example.

File: draft-ietf-dnsop-svcb-https.md (hunk 2 above)
Comment (bemasc) [reply draft-ietf-dnsop-svcb-https.md:1223]: Yes, it does.  I added this to the last sentence of this example.

File: draft-ietf-dnsop-svcb-https.md [hunk 3]
```diff
@@ -1212,12 +1212,17 @@ Consider a simple zone of the form:
...
+to indicate that https://simple.example supports QUIC
 in addition to HTTPS over TCP (an implicit default).
-The record could also include other information
-(e.g. non-standard port, ECH configuration).
+The record could also include other information (e.g. non-standard port,
+ECH configuration).  For https://simple.example:8443, the record would be:
+
+    _8443._https 7200 IN HTTPS 1 . alpn=h3
+
+These records also tell clients to replace the scheme with "https" when
```
Comment (enygren) [inline draft-ietf-dnsop-svcb-https.md:1224]:
```suggestion
These records also respectively tell clients to replace the scheme with "https" when
```

**Pull request merged**